site/server/svgPngExport.tsx

```diff
@@ -10,8 +10,13 @@ global.App = { isEditor: false }
 import { ChartConfig, ChartConfigProps } from "charts/ChartConfig"
 
 const svgoConfig: svgo.Options = {
-    floatPrecision: 1,
-    plugins: [{ collapseGroups: false }, { removeViewBox: false }]
+    floatPrecision: 2,
```

Comment:
Why switch to `2`? Did you spot anything that looked off?

Reply:
No, but the output size differs only by about .1%, so I guess we can go with 2 just as well.
The only visible difference was a text line that was about 1px off, but it didn't look off.